Cargo.toml

```diff
@@ -86,6 +81,14 @@ toml_edit = "0.22.11"
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
 typed-path = "0.9.1"
+uv-distribution-filename = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
+uv-distribution-types = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
+uv-install-wheel = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
+uv-pep440 = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
+uv-pep508 = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
+uv-platform-tags = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
+uv-pypi-types = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
+uv-requirements-txt = { git = "https://github.com/astral-sh/uv", tag = "0.4.28" }
```

Comment:
Should we group these with the other uv crates?

Reply:
Taplo does the formatting right?